packages/shared/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@booking-services/shared",
+  "version": "0.0.0",
```

Can we change this to `0.0.1`?
```suggestion
  "version": "0.0.1",
```